rinja_parser/src/target.rs

```diff
@@ -7,6 +7,60 @@ use crate::{
     bool_lit, char_lit, identifier, keyword, num_lit, path_or_identifier, str_lit, ws,
 };
 
+fn check_underscore<'a>(i: &'a str, target: &Target<'a>) -> Result<(), ParseErr<'a>> {
+    match target {
+        Target::Name("_") | Target::Placeholder(_) => {
+            return Err(winnow::error::ErrMode::Cut(ErrorContext::new(
+                "reserved keyword `_` cannot be used here",
+                i,
+            )));
+        }
```

Comment:
IMHO, this use of placeholders is fine. Maybe I want to `{% let _ = iter.next() %}` or `{% let [_] = slice.try_into().unwrap() %}`, because I want to test something in my code, generate an effect, whatever. Sorry, but I think whole function can the removed, because I don't want to restrict what can be skipped on the lhs of an assignment. :-/

Reply:
Ok, let's go then. :)